packages/block-editor/src/components/rich-text/content.scss

```diff
@@ -40,3 +36,18 @@ figcaption.block-editor-rich-text__editable [data-rich-text-placeholder]::before
 		background: rgb(255, 255, 0);
 	}
 }
+
+[data-rich-text-comment],
```

Comment:
What is this attribute about?

Reply:
It stores the data inside comments

Reply:
And also marks a placeholder as a comment